static/app/utils/gettingStartedDocs/getPlatformPath.spec.ts

```diff
@@ -49,34 +49,41 @@ describe('getPlatformPath', () => {
 
     // Frameworks
     expect(getPlatformPath(getFrameworkPlatformWithId('capacitor'))).toEqual(
-      'capacitor/capacitor'
+      'language/language'
+    );
+
+    expect(getPlatformPath(getFrameworkPlatformWithId('ionic'))).toEqual(
+      'language/language'
+    );
```

Comment:
Not sure how useful the "handles special cases" test is anymore 🤔 
imho we should substitute that with a test for the `platform.id === platform.language` logic branch.